wheatley/types.py

```diff
@@ -0,0 +1,54 @@
+""" A module to hold common type aliases """
+
+from typing import Any, Dict, NewType, List
+from wheatley.bell import Bell
+
+
+CallDef = NewType('CallDef', Dict[int, str])
+JSON = Dict[str, Any]
+Places = NewType('Places', List[int])
+Row = NewType('Row', List[Bell])
+
+
+class Stroke:
```

Comment:
I think this should be in a separate file as it is a class and not a type

Reply:
Just to note as a more functional change than the other tidyups of logic, this needs some manual testing

Reply:
I definitely agree with the testing.

But given free reign I would probably combine the `Bell` class into `types.py`.  However, putting different classes/structs into one file (even if they have similar functionality) is I think a very Rusty thing to do, so I'll go with whatever is more Pythonic or clearer.

Reply:
I know I lean more to the C# single class per file.

I guess a file named `types` seems reasonable for type aliases, but starts to feel rather generic once you start adding other assorted classes.

A 40 line class like this with some constants seems reasonable to have in it's own file. For bonus goodness it could even have its own test file.

Reply:
Actually can we please rename this file? The file name clashes with the module `types` so crashes Pycharm's debugger.

Maybe `aliases.py`?

Reply:
I think for consistency with the rest OOP we should split the files up.  I'll also rename `types.py` to `aliases.py`.  I just tend to lean more towards the Rust put-your-tests-in-the-same-file mentality (I've been writing a lot of Rust code recently), but doing that feels very unidiomatic in Python, so I'll make some more tests in the `tests` folder.

Reply:
- [x] File renamed
- [ ] `Stroke` moved
- [ ] Tests added for `Bell` and `Stroke`

Reply:
Done!